packages/amplify-cli/src/extensions/amplify-helpers/auth-notifications.ts

```diff
@@ -0,0 +1,92 @@
+import { $TSAny, $TSContext, stateManager, exitOnNextTick, FeatureFlags, pathManager } from 'amplify-cli-core';
+import { collectDirectivesByType, readProjectConfiguration } from 'graphql-transformer-core';
+import { printer, prompter } from 'amplify-prompts';
+import { DocumentNode, FieldNode, DirectiveNode, parse } from 'graphql';
+import path from 'path';
+import fs from 'fs-extra';
+
+async function flipNotificationFlag(projectPath: string, flagName: string, value: boolean): Promise<void> {
+  let config = stateManager.getCLIJSON(projectPath, undefined, {
+    throwIfNotExist: false,
+    preserveComments: true,
+  });
+
+  config.features.graphqltransformer[flagName] = value;
+  stateManager.setCLIJSON(projectPath, config);
```

Comment:
We should either throw if the file doesn't exist, or we need better handling in the event that config is undefined.